app/utilities/schema.py

```diff
@@ -177,20 +191,45 @@ def load_schema_from_url(schema_url, language_code):
 
     constructed_schema_url = f"{schema_url}?language={language_code}"
 
-    req = requests.get(constructed_schema_url)
-    schema_response = req.content.decode()
-    response_duration_in_milliseconds = req.elapsed.total_seconds() * 1000
+    session = requests.Session()
```

Comment:
This will need revisiting in future, we most likely want to reuse a single session. Session already handles connection pooling for us.